src/modules/nodes/nodeQueries.ts

```diff
@@ -219,7 +221,12 @@ const fetchNodeTree = async ({
 }: NodeTreeGetParams): Promise<NodeTree> => {
   const [root, children] = await Promise.all([
     fetchNode({ id, language, taxonomyVersion }),
-    fetchChildNodesWithArticleType({ id, language, taxonomyVersion }),
+    fetchChildNodesWithArticleType({
+      id,
+      language,
+      nodeType: [`${TOPIC_NODE},${RESOURCE_NODE}`],
```

Comment:
Denne er kanskje litt lett å misforstå. Forstår jeg rett i at syntaksen tax forventer er `nodeType=[TOPIC,RESOURCE]`? Isåfall er jeg mer fan av at dette er et vanlig array ([TOPIC, RESOURCE]) istedenfor et array som alltid har et element, for deretter å endre det i fetch-funksjonen.